iphone/cli/hooks/run.js

```diff
@@ -13,14 +13,24 @@ const appc = require('node-appc'),
 	ioslib = require('ioslib'),
 	i18n = appc.i18n(__dirname),
 	__ = i18n.__,
-	__n = i18n.__n;
+	__n = i18n.__n,
+	exec = require('child_process').exec;
+
 
 exports.cliVersion = '>=3.2';
 
 exports.init = function (logger, config, cli) {
 	cli.addHook('build.post.compile', {
 		priority: 10000,
 		post: function (builder, finished) {
+			if (cli.argv.target === 'mac') {
+				logger.info(__('Launching Mac'));
+				const command = 'open -a ' + builder.iosBuildDir + '/' + builder.tiapp.name + '.app/Contents/MacOS/'+ builder.tiapp.name;
```

Comment:
<!--
  1 failure:  iphone/cli/hooks/...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/hooks/run.js;
  Line: 28;
-->
- :no_entry_sign: iphone/cli/hooks/run.js line 28 – Operator '+' must be spaced. (space-infix-ops)